app/models/comment.php

```diff
@@ -20,17 +20,116 @@ public static function getComments($offset, $limit, $id)
         $db = DB::conn();
         $rows = $db->rows("SELECT * FROM comment 
                         WHERE thread_id = ? ORDER BY created ASC LIMIT {$offset}, {$limit}", array($id));
```

Comment:
i suggest to check also if the `$offset` and `$limit` is integer or not :bow: 
you may use any these if you want :grin:
```php
is_int()
ctype_digit()
$offset = intval($offset)
$limit = (int)$limit;
```